src/electron/views/automated-checks/test-view.tsx

```diff
@@ -18,15 +21,23 @@ export type TestViewProps = {
     scanMetadata: ScanMetadata;
     cardsViewData: CardsViewModel;
     userConfigurationStoreData: UserConfigurationStoreData;
-};
+} & HeaderSectionProps;
```

Comment:
We usually don't add props like this; it's probably better to enumerate to be clear.

However, I'd say that we should probably just make the props the content page info rather than splitting it apart? Unless I'm missing something, seems unnecessary to me.